app/managers.py

```diff
@@ -0,0 +1,43 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+        self._table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self._table_name} "
+            f"(first_name,last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> list[Actor]:
+        actors_rows = self._connection.execute(
+            f"SELECT * FROM {self._table_name}"
+        )
+        return [Actor(*row) for row in actors_rows]
+
+    def update(self, id_upd: int,
+               f_name_upd: str,
+               l_name_upd: str) -> None:

```

Comment:
These variable names are not very good. If you don't want to use three whole words, you may name them something like `actor_id`, `first_name`, `last_name`.
It's not necessary to duplicate method name in the argument names.